test/jdk/java/awt/Dialog/GetLocationTest_1.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (c) 1999, 2024, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import java.awt.Color;
+import java.awt.Component;
+import java.awt.Dialog;
+import java.awt.Frame;
+import java.awt.Graphics;
+import java.awt.Point;
+import java.awt.Rectangle;
+import java.awt.Window;
+import java.awt.event.MouseAdapter;
+import java.awt.event.MouseEvent;
+
+/*
+ * @test
+ * @bug 4168481
+ * @summary Test to verify Dialog getLocation() regression on Solaris
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
+ * @run main/manual GetLocationTest_1
+ */
+
+public class GetLocationTest_1 {
+    public static void main(String[] args) throws Exception {
+        String INSTRUCTIONS = """
+                1. Click in in the blue square and the yellow window should come
+                   up with the top left by the cursor
+                2. If you see this correct behavior press PASS. If you see that
+                   the yellow window is offset by an inset (5, 24) press FAIL
```

Comment:
how will tester knows that the inset is of this dimension, it might vary from platform to platform and also vary with different scaling? maybe just delete this hardcoded dimension?

Reply:
Updated.